prow/config/jobs/metal3-io/ip-address-manager.yaml

```diff
@@ -0,0 +1,180 @@
+presubmits:
+  metal3-io/ip-address-manager:
+  - name: gomod
+    skip_if_only_changed: '(((^|/)OWNERS)|((^|/)OWNERS_ALIASES)|(\.md))$'
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/gomod.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: docker.io/golang:1.22
+        imagePullPolicy: Always
+  # NOTE: The test jobs are for verifying the Makefile and hack/* scripts only
+  - name: test
+    run_if_changed: "^(Makefile|hack/.*)$"
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - test
+        command:
+        - make
+        image: docker.io/golang:1.22
+        imagePullPolicy: Always
+  - name: markdownlint
```

Comment:
Can we also split these? Ideally each branch would be complete on its own.

Reply:
It can be done. Can we do it separately? I already started applying this and it looks like it is working :slightly_smiling_face: